exercises/crypto-square/description.md

```diff
@@ -22,10 +22,10 @@ is normalized to:
 "ifmanwasmeanttostayonthegroundgodwouldhavegivenusroots"
 ```
 
-The plaintext should be organized in to a rectangle.  The size of the
+The plaintext should be organized in to a rectangle or square.  The size of the
```

Comment:
That issue does show that c and r can be as small as possible 
But unless I miss something it does not say that squares are different to rectangles.

Reply:
Because i have struggled to solve this rust puzzle sincei thought c and r should be different, to the point i submitted an issue because i was unable to solve it. 
It seems to be clear for everyone but me that when you are refering to a rectangle the first thing that comes to mind is a square. Sorry for my stupidity. 
I thought exercism was about "empathetic conversations". That was not what i had in mind.

Reply:
Nobody's calling anybody stupid!

This description is far from ideal, and I agree that it needs to be updated to mention that `c` needs to be as small as possible. We were just questioning the second change.

I understand if you don't want to continue working on this because of the frustration. I wouldn't want the whole problem to be forgotten because of that, so I opened a new PR to fix the remaining problem with the description: https://github.com/exercism/problem-specifications/pull/1859/files

Reply:
@angelikatyborska @wolf99 the original paper that inspired this problem says ```One classic method for composing secret messages is called a square code.  The spaces are removed from the english text
and the characters are written into a square (or rectangle). ```

Why do you think the problem is more interesting if you remove the hint that a square is also a rectangle ?

(Nobody directly called me stupid, but when someone is telling you that he has been confused, because he didn't thought while solving a programing problem about rectangles, that a square was a special case of a rectangle, telling him "But unless I miss something it does not say that squares are different to rectangles?" or "Why is it misleading if all squares are by definition rectangles?" is rather insulting.)